central/deployment/datastore/datastore_test_constructors.go

```diff
@@ -78,6 +100,25 @@ func GetTestPostgresDataStore(t testing.TB, pool postgres.DB) (DataStore, error)
 	clusterRanker := ranking.ClusterRanker()
 	namespaceRanker := ranking.NamespaceRanker()
 	deploymentRanker := ranking.DeploymentRanker()
+	mockCtrl := gomock.NewController(t)
```

Comment:
Same as before, avoid mocking datastores inside `GetTestPostgresDataStore` functions as callers usually expect these to be real datastores.